index.html

```diff
@@ -303,6 +303,32 @@ <h2 class="refer-link">LET'S CONNECT</h2>
                 </div>
             </div>
         </section>
-       </main>
+        <section id="contact-container">
+            <div id="contact-items">
+                <div id="description-contact-box">
+                    <h1 class="contact-title-1">Contact me</h1>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![sssa](https://user-images.githubusercontent.com/69638013/159566069-d8403d0e-1347-4858-becd-9ce31698e464.png)  | ![Screenshot_24](https://user-images.githubusercontent.com/69638013/159566045-20397890-a6f7-4ab7-beab-a19a3f03bca9.png)

- [ ] Kindly change this h1 tag to any hx tag from h2-h6, It is better for your page to have just one h1 tag for semantic purposes
- [ ] Kindly add the appropriate styling before the "contact me" header, there should be a spacing please check the Figma design